src/index.js

```diff
@@ -1,14 +1,30 @@
 import './style.css';
-import domElement from './dom.js';
+import domItems from './dom.js';
+import { getScores, postScores } from './api-Call.js';
 
-const dom = document.querySelector('.data');
+const GAME_ID = 'aJt95NEpxjlxmmqFfIJL';
 
-const domItems = () => {
-  domElement.forEach((e) => {
-    const li = document.createElement('li');
-    li.innerHTML = `${e.name}:${e.score}`;
-    dom.appendChild(li);
+const refresh = document.getElementById('refresh');
+const form = document.getElementById('form');
+
+refresh.addEventListener('click', () => {
+  getScores(GAME_ID).then((scores) => {
+    domItems(scores.result);
   });
-};
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const body = {
+    user: document.getElementById('user').value,
+    score: parseInt(document.getElementById('score').value, 10),
+  };
+
+  postScores(GAME_ID, body);
```

Comment:
- [OPTIONAL] I would suggest adding some validation to the inputs before calling the `postScores` function. Currently, we get a response with `400` status code bad request from the server when submitting wrong inputs.

![Screenshot from 2021-11-19 00-21-50](https://user-images.githubusercontent.com/40503704/142512318-20bc2816-49d4-4142-b596-92059f8f0b80.png)